module2/homework/insensitivePalindrom/palindrome.cpp

```diff
@@ -0,0 +1,10 @@
+#include "palindrome.hpp"
+#include <algorithm>
+
+bool is_palindrome(std::string str)
+{
+    str.erase(std::remove_if(str.begin(), str.end(), [](char letter) { return !(std::isalpha(letter)); }), str.end());
+    std::transform(str.begin(), str.end(), str.begin(), [](auto letter) { return std::tolower(letter); });
```

Comment:
This is for example good usage of std::transform. Because here you modify the container.